specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json

```diff
@@ -3103,6 +3223,236 @@
         }
       ],
       "description": "Represents set of actions written in SQL language-based syntax that is performed against a ServiceBus.Messaging.BrokeredMessage "
+    },
+    "PremiumMessagingRegions": {
+      "properties": {	  
+        "properties": {
+		"x-ms-client-flatten": true,
```

Comment:
Could have made more sense using ```x-ms-client-flatten``` if ```PremiumMessagingRegions``` were a request parameter, since most of its properties are readonly, this feels redundant. Also, please verify the structure of this model looks; unnaturally convoluted.